### PR TITLE
Custom header for SDK Version Info

### PR DESCRIPTION
**Pull request by MarcoGix**

### CODE REVIEW

File: Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
+using System;
+
 namespace Artesian.SDK.Service
 {
     /// <summary>
     /// Artesian Constants
     /// </summary>
     public abstract class ArtesianConstants
     {
+        internal const string SDKVersion = "v2.2.1";
```
Comment (AndreaCuneo) [inline Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs:15]: this 'version' should be linked to the packageVersion of nuget, not a separate string. Should be somewhere in the Assembly

File: Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs [hunk 2]
```diff
@@ -2,14 +2,24 @@
...
+
 namespace Artesian.SDK.Service
 {
     /// <summary>
     /// Artesian Constants
     /// </summary>
     public abstract class ArtesianConstants
     {
-        internal const string CharacterValidatorRegEx = @"^[^'"",:;\s](?:(?:[^'"",:;\s]| )*[^'"",:;\s])?$";
+        internal const string SDKVersion = "v2.2.1";
```
Comment (AndreaCuneo) [inline Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs:16]: remove this line, misleading

File: Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs [hunk 3]
```diff
@@ -2,14 +2,24 @@
...
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+		public static string SDKVersionHeaderValue = $@".NET<{Assembly.GetExecutingAssembly().GetName().Version}>,{Environment.OSVersion.Platform}<{Environment.OSVersion.Version}>,{PlatformServices.Default.Application.RuntimeFramework.Identifier}<{PlatformServices.Default.Application.RuntimeFramework.Version}>";
```
Comment (AndreaCuneo) [inline Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs:19]: the format is not with <> afaik, ask Alessandro. It was changed during development, but he probably forgot to update the Task/Requirement specification. @sicurezza

Comment (AndreaCuneo) [inline Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs:19]: The original code had also a 'v' in front of the version number. Not sure that is the expected convention. Ask @sicurezza if is v1.2.3 or just 1.2.3. 

If unspecified, please specify it and make sure existing implementations uses the chosen format.

**Review by AndreaCuneo — CHANGES_REQUESTED** [review]

File: Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs (hunk 3 above)
Comment (AndreaCuneo) [reply Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs:19]: also is missing 'readonly' specifier

Comment (MarcoGix) [reply Artesian/Artesian.SDK/Service/Configuration/ArtesianConstants.cs:19]: Thanks, now I have the right version

**Review by AndreaCuneo — APPROVED** [review]

**Pull request merged**